cmd/kindnetd/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Kubernetes Authors.
+Copyright 2016 The Kubernetes Authors.
```

Comment:
this seems to shift the year back to 2016.

Reply:
oops, thanks! this commit started as copying the changes from the last PR 😉